### DISCUSSION TITLE
How to automate the deployment of ESXI systems and automatically configure fixed IP addresses for hosts through Cobbler

### DISCUSSION THREAD

**Discussion by wangxu186189** [cobbler/cobbler#3546] [Version specific questions]
How to automate the deployment of ESXI systems and automatically configure fixed IP addresses for hosts through Cobbler

**Comment by SchoolGuy** [comment]
I have never actually done this but the process should be the same as with other distros:

1. `cobbler import <iso>`
2. `cobbler mkloaders`
3. `cobbler sync`

And that is about it for deploying ESXI itself. For systems hosted through ESXI, you need to make Cobbler reachable by the ESXI hosts. After that it behaves like a normal libvirt VM where the DHCP hands out the next server and after that the well-known netbooting process by grub/pxelinux/ipxe is taking over.
